packages/datatrak-web/src/features/Tasks/TasksTable/AssignTaskModal.tsx

```diff
@@ -5,22 +5,92 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
 import { useForm, Controller } from 'react-hook-form';
+import { Typography } from '@material-ui/core';
 import { Modal, ModalCenteredContent } from '@tupaia/ui-components';
 import { AssigneeInput } from '../AssigneeInput';
 import { useEditTask } from '../../../api';
 import { Task } from '../../../types';
+import { useRepeatScheduleOptions } from '../CreateTaskModal/RepeatScheduleInput';
+import { StatusPill } from '../StatusPill';
+import { displayDate } from '../../../utils';
 
 const Container = styled(ModalCenteredContent)`
-  width: 20rem;
+  width: 26rem;
   max-width: 100%;
   margin: 0 auto;
+  padding-block: 2.5rem;
+`;
+
+const MetaDataContainer = styled.div`
+  padding-inline: 1rem;
+  padding-block-start: 1.1rem;
+  padding-block-end: 1.5rem;
+  border: 1px solid ${({ theme }) => theme.palette.divider};
+  border-radius: 4px;
+  margin-block-end: 1.2rem;
+`;
+
+const Title = styled(Typography).attrs({
+  variant: 'h3',
+})`
+  font-size: 0.875rem;
+  color: ${({ theme }) => theme.palette.text.secondary};
+  font-weight: normal;
+  margin-block-end: 0.2rem;
+`;
+
+const Value = styled(Typography)`
+  font-size: 0.875rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const ItemWrapper = styled.div`
+  &:not(:last-child) {
+    margin-block-end: 1.2rem;
+  }
+`;
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  &:first-child {
+    width: 58%;
+    padding-inline-end: 1rem;
+  }
+  &:last-child {
+    width: 42%;
+    border-left: 1px solid ${({ theme }) => theme.palette.divider};
+    padding-inline-start: 1rem;
+  }
+  ${ItemWrapper} {
+    height: 2.5rem;
+  }
+`;
+
+const Row = styled.div`
+  display: flex;
+  margin-block-end: 1.2rem;
 `;
 
 interface AssignTaskModalProps {
   task: Task;
   Button: React.ComponentType<{ onClick: () => void }>;
 }
 
+const useDisplayRepeatSchedule = (task: Task) => {
+  // TODO: When repeating tasks are implemented, make sure the repeat schedule is displayed correctly once a due date is returned with the task
+  const repeatScheduleOptions = useRepeatScheduleOptions(task?.dueDate);
```

Comment:
this is pretty minor but I don't think that useRepeatScheduleOptions needs to be a hook (ie. it could be named getRepeatScheduleOptions and so useDisplayRepeatSchedule would then also not need to be a hook.